javaForms/Dockerfile

```diff
@@ -41,8 +41,8 @@ RUN apt-get -q -y update && \
     msopenjdk-21 \
     # ADD YOUR PACKAGES HERE
 # DO NOT REMOVE THIS LABEL: this is used for VS Code automation
-    # __torizon_packages_dev_start__
-    # __torizon_packages_dev_end__
+    # __torizon_packages_build_start__
+    # __torizon_packages_build_end__
```

Comment:
the string here mention `build` but on the `torizonPackages.json` it's mention "devRuntimeDeps", it does not maintain consistency.

Reply:
Not sure I understood this one

Reply:
like, the `__torizon_packages_dev_start__` is not build? Something is not "glueing" the name on the JSON and here on the headers.

Reply:
Before this commit it was `__torizon_packages_dev_start__` for the debug runtime container and also for the build/compilation containers (Dockerfile.sdk and build part of Dockerfile). Now  `__torizon_packages_dev_start__` is just for the debug runtime container (`devRuntimeDeps`) and `__torizon_packages_build_start__` for the build/compilation containers (`buildDeps`)